app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SessionsController < ApplicationController
+
+  def new
+  end
+
+  def create
+    @user = User.find_by(email: params[:email])
+    if @user.authenticate(params[:password])
+      session[:user_id] = @user.id
+      redirect_to '/'
+    else
+      redirect_to new_sessions_path
+    end
+  end
+
+  def destroy
+    session[:user_id] = nil
+
+    redirect_to '/'
+  end
+
```

Comment:
it would be nice to wrap the form params in a session_params method for security!